appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { MockApi } from 'api-mocks/mock';
+import {
+  KeysScreen,
+  SetupKeyScreen,
+  SplashScreen,
+} from '../../../screenobjects/all-screens';
+import { ekmKeySamples } from "../../../../api-mocks/apis/ekm/ekm-endpoints";
+import { CommonData } from "../../../data";
+import BaseScreen from "../../../screenobjects/base.screen";
+import AppiumHelper from "../../../helpers/AppiumHelper";
+
+describe('SETUP: ', () => {
+
+  it('EKM invalid key update error handled gracefully', async () => {
```

Comment:
So you tried to make specific test file for this test?

Reply:
I haven't found exact reason for fail, but when it fails - it happens on 3rd app restart.
Maybe app isn't fully unloaded in this case and it leads to app crash. After test split it passes well each time

Reply:
OK, got it